server/controller.js

```diff
@@ -0,0 +1,16 @@
+// controllers for requests to server
+const Recommendation = require('../database/index.js');
+
+
+const getAllData = (req, res, data) => {
+  // get all data from db and send to client
+  Recommendation.find({})
+    .then(() => {
```

Comment:
Not sure if this is an issue you're running into, but when I use find() it returns an array containing my data, so to get what I want I use `data[0]`

Reply:
That might help. Also I realized I wasn't creating a connection to the database so this function does not work yet. I will fix this after I get S3 set up.